manager/src/main/java/org/openremote/manager/setup/builtin/ManagerDemoSetup.java

```diff
@@ -20,13 +20,15 @@
 package org.openremote.manager.setup.builtin;
 
 import org.apache.commons.io.IOUtils;
+import org.openremote.agent.protocol.dmx.artnet.ArtnetClientProtocol;
```

Comment:
We don't want to provision `Artnet` protocol in our demo setup as it doesn't make sense, don't mind it here for dev purposes but will need to be removed from here....we don't provision protocols in demo as connections won't work